sql_scripts/3_cities.sql

```diff
@@ -0,0 +1,567 @@
+CREATE TABLE IF NOT EXISTS cities
+(
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(50) NOT NULL,
+    state_id INTEGER  NOT NULL 
+        references states (id)
+);
+
+
+INSERT INTO cities(name,state_id) VALUES ('Minneapolis',31);
```

Comment:
Please refer to [sql_scripts/2_states.sql] and make same changes for these insert stataments